### PR TITLE
Add milestone report points into readme

### PR DESCRIPTION
**Pull request by shenshan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
-+ See the [workflow-imaging](https://github.com/datajoint/workflow-imaging) and [workflow-ephys](https://github.com/datajoint/workflow-ephys) repositories for example usages of `elements-lab`.
++ `element-lab` is not a complete workflow by itself, but rather a modular design of tables and dependencies that can be flexibly attached to any DataJoint workflow.
++ See the [workflow-imaging](https://github.com/datajoint/workflow-imaging) and [workflow-ephys](https://github.com/datajoint/workflow-ephys) repositories for example usages of `element-lab`.
 
 + See the [datajoint-elements](https://github.com/datajoint/datajoint-elements) repository for a detailed description of the DataJoint elements and workflows.
 
 ## Element architecture
 
-![elements lab diagram](images/elements_lab_diagram.svg)
+![element lab diagram](images/elements_lab_diagram.svg)
```
Comment (kabilar) [inline README.md:14]:
```suggestion
![element lab diagram](images/element_lab_diagram.svg)
```

Comment (kabilar) [reply README.md:14]: Will have to rename the file as well.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,14 @@
...
-+ `elements-lab` is not a complete workflow by itself, but rather a modular design of tables and dependencies. 
-
-+ `elements-lab` can be flexibly attached to any DataJoint workflow.
+Element Lab contains lab related information, such as Lab, User, Project, Protocol, Source.
 
 ## Element usage
 
-+ See the [workflow-imaging](https://github.com/datajoint/workflow-imaging) and [workflow-ephys](https://github.com/datajoint/workflow-ephys) repositories for example usages of `elements-lab`.
++ `element-lab` is not a complete workflow by itself, but rather a modular design of tables and dependencies that can be flexibly attached to any DataJoint workflow.
++ See the [workflow-imaging](https://github.com/datajoint/workflow-imaging) and [workflow-ephys](https://github.com/datajoint/workflow-ephys) repositories for example usages of `element-lab`.
```
Comment (kabilar) [inline README.md:8]:
```suggestion
+ See the [workflow-calcium-imaging](https://github.com/datajoint/workflow-calcium-imaging) and [workflow-array-ephys](https://github.com/datajoint/workflow-array-ephys) repositories for example usages of `element-lab`.
```

File: Background.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+## Description of modality, user population
+Most pipelines track some information about the lab, including the facilities, experiment rigs, and users. All interviewed labs have some version of these elements. They also have custom interfaces and GUIs for entering.
```
Comment (kabilar) [inline Background.md:2]:
```suggestion
Most pipelines track some information about the lab, including the facilities, experiment rigs, and users. All interviewed labs have some version of these elements. They also have custom interfaces and GUIs for data entry.
```

File: README.md [hunk 4]
```diff
@@ -1,17 +1,14 @@
 # DataJoint Element - Lab
```
Comment (kabilar) [inline README.md:1]: Should we provide a link to the `Background.md` file in the `README.md`?

File: Background.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
++ International Brain Laboratory https://github.com/int-brain-lab/IBL-pipeline
++ BrainCoGs (Princeton Neuroscience Institute) https://github.com/BrainCOGS/U19-pipeline_python; https://github.com/BrainCOGS/U19-pipeline-matlab
++ MoC3 (Columbia Zuckerman Institute)
+    + Churchland Lab: https://github.com/ZuckermanBrain/datajoint-churchland/tree/master/churchland_pipeline_python
+    + Costa Lab (private repository)
+    + Hillman Lab: https://github.com/ZuckermanBrain/datajoint-hillman
+
+## Development and validation
+Through our interviews and direct collaboration on the precursor projects, we identified the common motifs in the lab schemas to create the Lab Management Element.
+This element works for diverse downstream pipelines and is always used in combination with other elements for specific experiments. As such it is validated jointly with the acquisition elements such as the Neuropixels Element and Calcium Imaging Element.
```
Comment (kabilar) [inline Background.md:17]:
```suggestion
This element works for diverse downstream pipelines and is always used in combination with other elements for specific experiments. As such, it is validated jointly with the acquisition elements such as the [Extracellular Array Electrophysiology Element](https://github.com/datajoint/elements-array-ephys) and [Calcium Imaging Element](https://github.com/datajoint/elements-calcium-imaging).
```

File: Background.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+## Precursor projects and interviews
+Over the past few years, several labs have developed DataJoint-based pipelines for lab management. Our team collaborated with several of them during their projects. Additionally, we interviewed these teams to understand their experiment workflow, associated tools, and interfaces.
+These teams include
+Team / Institution / Pipeline repository
++ International Brain Laboratory https://github.com/int-brain-lab/IBL-pipeline
++ BrainCoGs (Princeton Neuroscience Institute) https://github.com/BrainCOGS/U19-pipeline_python; https://github.com/BrainCOGS/U19-pipeline-matlab
++ MoC3 (Columbia Zuckerman Institute)
+    + Churchland Lab: https://github.com/ZuckermanBrain/datajoint-churchland/tree/master/churchland_pipeline_python
+    + Costa Lab (private repository)
+    + Hillman Lab: https://github.com/ZuckermanBrain/datajoint-hillman
```
Comment (kabilar) [inline Background.md:13]:
```suggestion
These teams include:
+ [International Brain Laboratory](https://github.com/int-brain-lab/IBL-pipeline) 
+ BrainCoGs (Princeton Neuroscience Institute), [Python pipeline](https://github.com/BrainCOGS/U19-pipeline_python), [MATLAB pipeline](https://github.com/BrainCOGS/U19-pipeline-matlab)
+ MoC3 (Columbia Zuckerman Institute)
    + [Churchland Lab](https://github.com/ZuckermanBrain/datajoint-churchland/tree/master/churchland_pipeline_python)
    + Costa Lab (private repository)
    + [Hillman Lab](https://github.com/ZuckermanBrain/datajoint-hillman)
```

**Review by kabilar — CHANGES_REQUESTED** [review]
Great work.  Few minor fixes suggested.

File: README.md (hunk 4 above)
Comment (shenshan) [reply README.md:1]: Good point.

**Review by dimitri-yatsenko — APPROVED** [review]

**Review by ttngu207 — APPROVED** [review]

**Review by kabilar — APPROVED** [review]

**Pull request merged**